core/flyout.js

```diff
@@ -328,6 +473,10 @@ Blockly.Flyout.prototype.scrollToStart = function() {
  * @private
  */
 Blockly.Flyout.prototype.wheel_ = function(e) {
+  // Don't scroll sideways.
```

Comment:
Why not?  Many mice have wheels that can scroll in the horizontal plane.  For horizontal scrolling use e.deltaX.